modules/token-holders/pages/seed-round.adoc

```diff
@@ -0,0 +1,232 @@
+= Seed round investors token distribution
+:experimental:
+// Define unicode for Apple Command key.
+:commandkey: &#8984;
+:IC: Internet Computer
+:company-id: DFINITY
+:sdk-short-name: DFINITY Canister SDK
+:sdk-long-name: DFINITY Canister Software Development Kit (SDK)
+ifdef::env-github,env-browser[:outfilesuffix:.adoc]
+
+If you participated in the DFINITY Foundation seed round by making a contribution in February 2017, now is the time to prepare for your token distribution.
+
+To receive your tokens, you need to complete the following steps:
+
+* Add an identity for your Ethereum-based address to the {IC}.
+* Create the neuron stake for your account.
+* Dissolve the neuron stake to view and access ICP tokens.
+* Complete the Know Your Customer (KYC) verification process.
+
+== Add an identity for your address to the {IC}
+
+As a participant in the seed round, you received a 12-word mnemonic seed phrase that was created with a DFINITY Chrome extension. 
+To receive your tokens, you need to use this 12-word mnemonic seed phrase to generate a public/private key pair for your identity that can be recognized by the {IC}.
+The output from your seed phrase is a security certificate in the form of a PEM file. 
+You can then import the PEM file using the {sdk-short-name} command-line interface, `dfx`, to create an {IC} *principal* for your identity. 
+A principal similar to a Bitcoin or Ethereum address and can be used to securely identify your account for making transactions and creating neurons.
+
+=== Access a secure device 
+
+To get start, you should first verify that you have access to a hardware wallet or an air-gapped computer that is secure and not connected to the internet.
```

Comment:
IMO, `Access a secure device` should be a **strong suggestion** to the user. It is not a technical blocker to continue the following steps. The users may feel ok to just use a normal pc.

And I would suggest to remove `hardware wallet` here because we haven't had support for it yet. When we support theat in the future, we may also need to add an instruction for that workflow. The `secure device` here currently only points to `air-gapped computer`.